listenbrainz/tests/unit/test_listen.py

```diff
@@ -100,3 +100,37 @@ def test_to_influx(self):
         self.assertEqual(data['fields']['artist_name'], listen.data['artist_name'])
 
         self.assertIn('inserted_timestamp', data['fields'])
+
+
+    def test_from_json(self):
+        json_row = {
```

Comment:
Could we possibly use `testdata/valid_single.json` here? (https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/testdata/valid_single.json). It would reduce the complexity and size of the test a lot.